minunit.h

```diff
@@ -190,6 +211,29 @@ static void (*minunit_teardown)(void) = NULL;
 	}\
 )
 
+#define mu_assert_string_eq(expected, result) MU__SAFE_BLOCK(\
+	const char* minunit_tmp_e;\
+	const char* minunit_tmp_r;\
+	minunit_assert++;\
+	if (!expected) {\
+		minunit_tmp_e = "<null pointer>";\
+	} else {\
+		minunit_tmp_e = (expected);\
```

Comment:
In general "expected" and "result" should not appear more than once in the body of the macro as they may be expressions and will be evaluated more than once if that is the case. I would initialize minunit_tmp_e and minunit_tmp_r to expected and result after the minunit_assert++; line like:

```
const char* minunit_tmp_e;\
...
minunit_assert++;\
minunit_tmp_e = (expected);\
...
if (!minunit_tmp_e) {\
    minunit_tmp_e = "<null pointer>"';\
}\
...
```